tests/e2e/specs/recommended-widget.spec.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import {
+	activatePlugin,
+	loginUser,
+	visitAdminPage,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { waitForWpAdmin } from '../utils';
+
+describe( 'Recommended widget', () => {
+	jest.setTimeout( 30000 );
+	it( 'Widget should be available', async () => {
+		await loginUser();
+		await activatePlugin( 'wp-parsely' );
+		await visitAdminPage( '/widgets.php', '' );
+
+		await waitForWpAdmin();
+
+		await page.keyboard.press( 'Escape' );
```

Comment:
Why is this `Escape` needed? If it's to close a modal pop-up, or something else, can we make this more descriptive - prepareWidgetScreen, closeNewWidgetScreenModal, etc.?